pkg/sfu/streamtrackermanager.go

```diff
@@ -133,12 +145,14 @@ func (s *StreamTrackerManager) AddTracker(layer int32) *streamtracker.StreamTrac
 
 	s.logger.Debugw("StreamTrackerManager add track", "layer", layer)
 	tracker.OnStatusChanged(func(status streamtracker.StreamStatus) {
-		s.logger.Debugw("StreamTrackerManager OnStatusChanged", "layer", layer, "status", status)
-		if status == streamtracker.StreamStatusStopped {
-			s.removeAvailableLayer(layer)
-		} else {
-			s.addAvailableLayer(layer)
-		}
+		s.layersNotifyOpQueue.Enqueue(func() {
```

Comment:
Sending this via ops queue as this messed up the available layers in `sfu.DownTrack`. There are locks here, but the callbacks happened out-of-order. As `sfu.DownTrack` is making a copy of the layers, it got stuck with incorrect data. Ensuring that it happens in order.

Actually, want to get rid of `availableLayers` notion fully as mentioned in PR commit notes above. Can be cleaned up at that time.